scripts/remove

```diff
@@ -64,6 +64,17 @@ ynh_remove_fpm_config
 
 ynh_remove_systemd_config
 
+#=================================================
+# REMOVE SERVICE FROM ADMIN PANEL
+#=================================================
+
+# Remove a service from the admin panel, added by `yunohost service add`
+if yunohost service status | grep -q $app
+then
+	echo "Remove $app service"
```

Comment:
`ynh_print_info` is not available in Yunohost 2.7.12? (requirements in manifest)

Reply:
No, we would need ynh 3 at least. Maybe even more than that... I don't really know when we have introduced that helper.

Reply:
https://github.com/YunoHost/yunohost/blob/stretch-unstable/debian/changelog#L185
I should have a look to my other PR...

Reply:
Ok, maybe in the helpers doc we should add the min version for each helper? (Could be horrible to maintain)

Reply:
That's a pretty good idea.
We're not adding that much helpers, but I agree that it's not going to be easy to maintain, considering we usually don't know which version it's going to be (do we ?)
Would you take care of that ? For the current helpers of course.

Reply:
When I have some time, I can do some archeology. I add it to my TODO list.

Reply:
As you want, otherwise, I can do it right now.

Reply:
Well, I can't for now, so if you want/can do it, otherwise i'll but maybe not today :-)

Reply:
Ok, as I'm still working on the official apps. Feeling more into that than the high_quality/featured tag today. I'll do it, it will be useful for my work.

Reply:
https://github.com/YunoHost/yunohost/pull/664